config-templates/salesforce/readme.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+This directory contains a set of config files for modeling salesforce data. It also includes several common metrics calcuated from salesforce leads and opportunities. If you see a metric that's missing please open an issue in this repo.
```

Comment:
nit: capitalize Salesforce
sp: calculated
Maybe remove "in this repo" since it's in the same repo as MetricFlow now?